### PR TITLE
Add loading and disabled binding

### PR DESCRIPTION
**Pull request by DiegoQuiros94**
## Summary ##

added action functions to set a loading icon while executing the action and to bind a button alpha component to the isEnabled value of the action

### CODE REVIEW

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Quality (Swiftlint): 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-reactive-core-ios/branches/add-loading-and-disabled-binding) for more details.

File: WolmoReactiveCore/Action.swift [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import MBProgressHUD
+import ReactiveSwift
+
+extension Action {
+    
+    /*
```
Comment (gmazzei) [inline WolmoReactiveCore/Action.swift]: It's just a detail but can we use `/**    */` instead of `/*     **/` as in the other files?

Comment (DiegoQuiros94) [reply WolmoReactiveCore/Action.swift]: sounds good

**Comment by woloxci** [comment]
**CodeStats metrics details:**

:white_check_mark: Code Quality (Swiftlint): 98.0


Click [here](http://codestats.wolox.com.ar/organizations/wolox/projects/wolmo-reactive-core-ios/branches/add-loading-and-disabled-binding) for more details.

**Pull request merged**